pkg/cmd/subcommand/install.go

```diff
@@ -211,14 +216,18 @@ func (i *Install) RunInstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 	operator.Inventory = inventoryPending
 	inventoryExist := getExistComponentsInventory(ctx, cl)
 
-	fmt.Fprintln(w, ti.BeforeTask("Start installing OpenFunction and its dependencies.\n"+
-		"Here are the components and corresponding versions to be installed:"))
+	util.BeforeTask("Start installing OpenFunction and its dependencies.\n" +
+		"Here are the components and corresponding versions to be installed:")
```

Comment:
Here are the components and corresponding versions to be installed:
=>
The following components will be installed: